broker-util/oo-accept-broker

```diff
@@ -160,8 +160,8 @@ function check_mongo_connectivity() {
       for mongo_replicant in $(echo $MONGO_HOST | sed s/,/\ /g)
       do
         ((replica_count++))
-        host=$(echo $mongo_replicant | cut -d: -f1)
-        port=$(echo $mongo_replicant | cut -d: -f2)
+        host=$(echo $mongo_replicant | cut -d: -f1 | tr -d '"')
+        port=$(echo $mongo_replicant | cut -d: -f2 | tr -d '"')
```

Comment:
Should probably be ` | tr -d \'\"` in case single-quotes are used.

Alternatively, what about using something like the following?
```shell
MONGO_HOST=$(ruby -r parseconfig -e 'puts ParseConfig.new("/home/mmasters/src/origin-server/broker/conf/broker.conf")["MONGO_HOST_PORT"].split(",").first')
```